types/cache-manager-fs-hash/package.json

```diff
@@ -6,7 +6,7 @@
         "https://github.com/rolandstarke/node-cache-manager-fs-hash#readme"
     ],
     "dependencies": {
-        "@types/cache-manager": "*"
+        "@types/cache-manager": "<4"
```

Comment:
What's funny is that this package (`cache-manager-fs-hash`) is actually out of date; it's latest version, 2.0, depends on `cache-manager@^5.5.1`.